exposee-ui/src/App.css

```diff
@@ -0,0 +1,42 @@
+#root {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+  text-align: center;
+}
+
+.logo {
```

Comment:
do you need all this logo boilerplate?

Reply:
i would be changing them. I got them from creating the vite package